rotkehlchen/data_import/importers/binance.py

```diff
@@ -317,6 +386,45 @@ def process_entry(
         importer.add_asset_movement(write_cursor=write_cursor, asset_movement=asset_movement)
 
 
+class BinanceDistributionEntry(BinanceSingleEntry):
+    AVAILABLE_OPERATIONS = (
+        'Cash Voucher Distribution',
+        'Mission Reward Distribution',
+    )
+
+    def process_entry(
+            self,
+            write_cursor: DBCursor,
+            importer: BaseExchangeImporter,
+            timestamp: Timestamp,
+            data: BinanceCsvRow,
+    ) -> None:
+        """
+        Process distributions from binance. It includes
+        Cash Voucher Distribution and Mission Reward Distribution. May raise:
+        - KeyError
+        - DeserializationError: if the event is malformed when being stored in the db
+        """
+        asset = data['Coin']
```

Comment:
no need to declare a variable here. Just use it directly in the constructor below
```suggestion
```